README.md

```diff
@@ -276,6 +276,25 @@ file, rename and edit it.
 <?php
 
 $settings = array (
+    // Security settings that must be set to avoid insecure SHA-1 usage.
```

Comment:
that block where signatureAlgorithm digestAlgorithm are described already appears down...